test-prof.gemspec

```
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
     "source_code_uri" => "http://github.com/test-prof/test-prof"
   }
 
-  spec.files = Dir.glob("lib/**/*") + Dir.glob("config/**/*") + Dir.glob("assets/**/*") + %w[README.md LICENSE.txt CHANGELOG.md]
+  spec.files = Dir.glob("lib/**/*") + Dir.glob("config/**/*") + Dir.glob("assets/**/*") + %w[README.md LICENSE.txt CHANGELOG.md] + %w[config/rubocop-rspec.yml]
```

`config/**/*` is already included
```suggestion
  spec.files = Dir.glob("lib/**/*") + Dir.glob("config/**/*") + Dir.glob("assets/**/*") + %w[README.md LICENSE.txt CHANGELOG.md]
```

---

Thanks! I've missed that completely.